requirements/base.in

```diff
@@ -0,0 +1,11 @@
+# Core requirements for using this package
+-c constraints.txt
+
+django
+Markdown
+django-sekizai
+django-mptt
+Pillow
+sorl-thumbnail
```

Comment:
Could you add to the description of the PR why this dependency was added? thes same for Pillow, six and bleach.

Please also sort them alphabetically.